test/agoneum/web/controllers/registration_controller_test.exs

```diff
@@ -1,30 +1,47 @@
 defmodule Agoneum.Web.RegistrationControllerTest do
   use Agoneum.Web.ConnCase
 
-  alias Agoneum.Account
+  describe "UI registration" do
```

Comment:
What would your thoughts be on splitting tests out by type (UI vs API), @mbramson @daveshah?

Im thinking add a `test/agoneum/web/controllers/api` directory and having another `registration_controller_test.exs` potentially. I feel like having both sets of tests in this file will eventually get cluttered